Sprint bente (#28)
SW814F20/Report: report/Sprint_2/Introduction.tex

> This chapter will highlight the progress we have made in our second sprint.

Reviewer: ```suggestion
This chapter will highlight the progress we made in our second sprint.
```

> The purpose of this sprint is to create, what we call \textbf{Screen}, which is a screen that the customer will create, using buttons, text and other widgets, we support.

Reviewer: I don't quite understand this. 
'what we call \textbf{Screen}'
Do you mean the Screen Editor?

Maybe this?:

```suggestion
The purpose of this sprint is to create, what we call a \textbf{Screen}, which is a screen that the customer will create, using buttons, text and other widgets, we support.
```

Author: Mente at vi har noget der hedder en screen, som i at der er Text, Button og andre widgets, og så har vi også en screen editor, som laver om på de her screens

These screens will be able to be translated into auto-generated tests, using a module we will create, that enables the developers to test the code they have written, to ensure that the functionalities the customer wants are supported in the software.

> Lastly, we want to focus on, is to understand the GitHub API, as we will use that, to create tasks for the developers.

Reviewer: ```suggestion
Lastly, we want to focus on, understanding the GitHub API, as we will use that, to create tasks for the developers.
```

The goals for this sprint is as follow:

\begin{itemize}
    \item Create Screen functionalities, such as Screen Editor and Screen Selector
    \item Begin work on a module to auto-generate tests

>     \item Research on using the GitHub API

Reviewer: ```suggestion
    \item Research the GitHub API
```
